src/api.ts

```diff
@@ -8,8 +8,9 @@ const parser: peg.Parser = require('./internal/parser');
 /**
  * Generates a MfmNode tree from the MFM string.
 */
-export function parse(input: string): MfmNode[] {
-	const nodes = parser.parse(input, { startRule: 'fullParser' });
+export function parse(input: string, opts?: Partial<{ fnNameList: string[]; }>): MfmNode[] {
+	opts = opts || { };
```

Comment:
Could use Typescript parameter default here:

```suggestion
export function parse(input: string, opts?: Partial<{ fnNameList: string[]; }> = {}): MfmNode[] {
```

Reply:
:cool: